packages/odyssey-react/src/components/FieldGroup/FieldGroup.module.scss

```diff
@@ -13,12 +13,12 @@
 .root {
   max-width: $max-line-length;
   margin-block-start: 0;
-  margin-block-end: $spacing-m;
+  margin-block-end: var(--SpaceEmM);
   padding-inline: 0;
   padding-block-start: 0;
-  padding-block-end: $spacing-m;
+  padding-block-end: var(--SpaceEmM);
   margin-inline: 0;
-  border-block-end: 1px solid $border-color-display;
+  border-block-end: 1px solid var(--ColorBorderDisplay);
```

Comment:
In other places, we're now offering full theming for all border props. I'm not sure it's needed, but we probably want to be consistent here.

For my part, I've called these `BorderWidth`, `BorderStyle`, and `BorderColor` in the local tokens.